dwave/preprocessing/presolve/cypresolve.pyx

```diff
@@ -27,6 +27,8 @@ from dimod.libcpp cimport ConstrainedQuadraticModel as cppConstrainedQuadraticMo
 from dimod.constrained.cyconstrained cimport cyConstrainedQuadraticModel, make_cqm
 from dimod.cyutilities cimport ConstNumeric
 
+from dwave.preprocessing.libcpp cimport Feasibility as cppFeasibility
```

Comment:
Why is this camel-backed?  Seems out of place, especially when used right next to "cpppresolver"

Reply:
It's a convention we've been using in Ocean to distinguish between the c++, Cython, and Python versions of various classes. In this case we could just use `Feasibility` because we don't (yet) have a Cython/Python equivalent, but it's useful for instance with `ConstrainedQuadraticModel`.